ArduinoCode/Pedal_Potentiometers/Pedal_Potentiometers.ino

```diff
@@ -88,10 +93,54 @@ void readSwitches() {
   brakePin1Val = digitalRead(brakePin1);
   brakePin2Val = digitalRead(brakePin2);
 
-  if ((brakePin1Val == LOW) && (brakePin2Val == LOW)) {
-    unsigned char valueMessage[8] = {1, 0, 0, 0, 0, 0, 0, 0};
-    CAN.sendMsgBuf(CAN_BRAKE, 0, 8, valueMessage); // send message that the switches are turned on
-  } else if (brakePin1Val != brakePin2Val) {
-    CAN.sendMsgBuf(CAN_BRAKE_BROKE, 0, 8, errorMessage); // send message that the two switch values are not equal
+  if ((brakePin1Val == LOW) || (brakePin2Val == LOW)) {  // switches are on
+    unsigned char valueMessage[1] = {1};
+    CAN.sendMsgBuf(CAN_BRAKE, 0, 1, valueMessage); // send message that the switches are turned on
+  } 
+  else {    // switches  are off
+    unsigned char valueMessage[1] = {0};
+    CAN.sendMsgBuf(CAN_BRAKE, 0, 1, valueMessage); // send message that the switches are turned off
   }
```

Comment:
This will need a few changes to add error messages: 
```suggestion
  if ((brakePin1Val == LOW) && (brakePin2Val == LOW)) {  // switches are on
    unsigned char valueMessage[1] = {1};
    CAN.sendMsgBuf(CAN_BRAKE, 0, 1, valueMessage); // send message that the switches are turned on
  } 
  else if ((brakePin1Val == HIGH) && (brakePin2Val == HIGH)) {  // switches are off
    unsigned char valueMessage[1] = {0};
    CAN.sendMsgBuf(CAN_BRAKE, 0, 1, valueMessage); // send message that the switches are turned off
  } else {
    // Send error message that switches do not agree
  }
```